### ISSUE TITLE
dashboard can't reject a paper under review

### ISSUE THREAD

**Issue by danielskatz** [openjournals/joss#659]
The dashboard gives me a link to a page for each submission.  If the paper is in pre-review, I can mark it as withdrawn or rejected, but if it is in review, I can only mark it as withdrawn, but not rejected. Why not? Sometimes we need to reject a paper where review has started.

**Comment by arfon** [comment]
Addressed in part by https://github.com/openjournals/whedon-api/pull/79 although the button still isn't there for papers under review.

**Comment by danielskatz** [comment]
I don't think we need this anymore as we handle this through editorialbot commands
